edx_exams/apps/core/templates/email/proctoring_attempt_rejected.html

```diff
@@ -0,0 +1,33 @@
+{% load i18n %}
+
+<p>
+    {% block introduction %}
+    {% blocktrans %}
+        Hello {{ username }},
+    {% endblocktrans %}
+    {% endblock %}
+</p>
+<p>
+    {% block status_information %}
+    {% blocktrans %}
+        Your proctored exam "{{ exam_name }}" in
+        <a href="{{ course_url }}">{{ course_url }}</a> was reviewed and the
+        course team has identified one or more violations of the proctored exam rules. Examples
+        of issues that may result in a rules violation include browsing
+        the internet, using a phone,
+        or getting help from another person. As a result of the identified issue(s),
+        you did not successfully meet the proctored exam requirements.
+    {% endblocktrans %}
+    {% endblock %}
+</p>
+<p>
+    {% block contact_information %}
+    {% blocktrans %}
+        To appeal your proctored exam results, please reach out with any relevant information
+        about your exam at 
+        <a href="{{contact_url}}">
+            {{ contact_url_text }}
```

Comment:
Is this email the correct place to escalate to? It's set up per provider, and I feel like it would make more sense for the course team to review this.

Reply:
this is a really good point. In my head this was the escalation email but that's not true, this email is just tech support for the tool. We don't save an escalation email for LTI but it sounds like we need one. I'm going to open a ticket to discuss adding this end to end.

For now I'm going to update this so this falls back on edx support in absence of a course team email.